<!DOCTYPE html>
<html>
  <head>
    <title>Research</title>
    
    <link rel="stylesheet" type="text/css" href="/css/main.css">
  
  </head>


  <body>
    <nav>
<ul>
<li><a href="/">Home</a></li>
<li><a href="/research">Research</a></li>
<li><a href="/members">Members</a></li>
<li><a href="/blog">Blog</a></li>
</ul>
    </nav>
<div class="container">

  		<div class="research">
			<h1>Research</h1>
			<p>Increasing evidences suggest that most of the mammalian genome is transcribed as RNAs. 
			However, not all of them are translated to become proteins. 
			The current estimate is that only few percent of the mammalian genome encodes for protein-coding genes.
			Up until recently, many of the remaining RNAs were considered as transcriptional noises and experimental errors 
			that do not have any functions. However, recent studies show that non-coding RNAs (ncRNAs) are functional 
			as in the case of microRNAs (miRs). 
			Besides miRs, there are more longer ncRNAs have been identified. 
			If their lengths are longer than 200 nucleotides (nt), these ncRNAs are collectively called “long non-coding RNAs (lncRNAs)”.
			Although lncRNAs are of great interested in the research community, their functions are largely unknown.
			This is especially true in the field of cardiovascular research, which we recently reviewed 
			(<a href="http://circres.ahajournals.org/content/116/4/737.abstract"><span class="ptHyperlink">Link</span></a>).
			<br>
			Given that genetic networks are ultimately controlled by epigenetic status of genes and 
			their promoter/enhancer regions, lncRNA-mediate recruitment might shed a light on the “missing link” 
			in the complete understanding of how genes are transcribed into proteins to perform functions 
			necessary for the development of organs and the differentiation of embryonic/adult stem cells 
			into tissue-specific cell types. </p><p class="ptNormal"></p>
			
			<p class="ptNormal">In last few years, we have developed a number of bioinformatics tools:  
			“<b>C-It</b>” (<a href="http://c-it.mpi-bn.mpg.de/"><span class="ptHyperlink">Link</span></a>) 
			for evolutionary-conserved, tissue-enriched, uncharacterized genes 
			
			“<b>SNP4Disease</b>” (<a href="http://snp4disease.mpi-bn.mpg.de/">
			<span class="ptHyperlink">Link</span></a>) for disease-related and/or -suspected SNPs
		
			“<b>Exon Array Analyzer</b> (<b>EAA</b>)” (<a href="http://eaa.mpi-bn.mpg.de/">
			<span class="ptHyperlink">Link</span></a>) for analyzing GeneChip Exon 1.0 ST Arrays (Affymetrix, Inc.)
			
			“<b>Gene Array Analyzer</b> (<b>GAA</b>)” (<a href="http://gaa.mpi-bn.mpg.de/">
			<span class="ptHyperlink">Link</span></a>) for analyzing GeneChip Gene 1.0 ST Arrays (Affymetrix, Inc.)
			“<b>noncoder</b>” (<a href="http://noncoder.mpi-bn.mpg.de/">
			<span class="ptHyperlink">Link</span></a>) for quantifying the expression changes of lncRNAs from GeneChip Exon 1.0 ST Arrays (Affymetrix, Inc.)
			
			
			Through the installment of benchtop next generation sequencer (Ion Torrent PGM), 
			we aim to further extend our efforts to build bioinformatics tools for scientific communities. 
			Our current focus is to elucidate the functions of lncRNAs 
			in the context of epigenetics using both dry and wet lab techniques.</p>
			
			<ul class="contacts">
				<li><a href="http://www.cardiovascular-regeneration.com">Institute of Cardiovascular Regeneration</a></li>
				</ul>
		</div>


</div>
  
  <footer>
   <ul>
   <li><a href="mailto:heart.lncrna@gmail.com">email</a></li>
   <li><a href="https://github.com/heartlncrna">github.com/heartlncrna</a></li>
   </ul>
  </footer>
  </body>

</html>

