<!DOCTYPE html>
<html>
  <head>
    <title>Research</title>
    <meta name="description" content="The home page of Shizuka Uchida's lab for cardiovascular bioinformatics.">    
    <meta name="keywords" content="bioinformatics, long non-coding RNA, lncRNA, lincRNA, heart, cardiovascular, cardiac, research, science, computational biology, molecular biology, RNA editing, RNA modification, RNA-seq">
    <script src="/js/analytics.js"></script>    
    <link rel="stylesheet" type="text/css" href="/css/main.css">
  
  </head>


  <body>
    <nav>
<ul>
<li><a href="/">Home</a></li>
<li><a href="/research/overview.html">Research</a></li>
<li><a href="/members/member_list.html">Members</a></li>
<li><a href="/publications/pub_list.html">Publications</a></li>
</ul>
    </nav>
<div class="container">

  		<div class="research">
			<h1>Research</h1>
			
			<p class="ptNormal">
			Increasing evidence suggests that most of the mammalian genome is transcribed as RNAs. 
			However, not all of them are translated to become proteins. 
			The current estimate is that only few percent of the mammalian genome encodes for protein-coding genes.
			Up until recently, many of the remaining RNAs were considered as transcriptional noises and experimental errors 
			that do not have any functions. However, recent studies show that non-coding RNAs (ncRNAs) are functional 
			as in the case of microRNAs (miRNAs). 
			Besides miRNAs, there are more longer ncRNAs have been identified. 
			If their lengths are longer than 200 nucleotides (nt), these ncRNAs are collectively called “long non-coding RNAs (lncRNAs)”.
			Although lncRNAs are of great interested in the research community, their functions are largely unknown.
			This is especially true in the field of cardiovascular research, which we recently reviewed 
			(<a href="http://circres.ahajournals.org/content/116/4/737.abstract"><span class="ptHyperlink">Link</span></a>) (<a href="https://www.liebertpub.com/doi/10.1089/ars.2017.7262"><span class="ptHyperlink">Link</span></a>) (<a href="https://www.frontiersin.org/articles/10.3389/fphys.2018.00522/full"><span class="ptHyperlink">Link</span></a>).
			</p>
			
			<p class="ptNormal">
Life on our planet depends on RNA. Despite our early understanding of RNA as an enabling messenger, it is not simply a static reflection of genetic activation. Indeed, the lifespan of RNA is more complex than previously thought. Upon transcription from the DNA template, RNA can be modified by various enzymes, which results in over 100 RNA modifications. Because of the potential involvement of RNA modifications in a variety of pathophysiologies, </b>epitranscriptomics</b> is quickly gaining momentum and our gaps in understanding epitranscriptomics requires further study. This is especially true in the field of cardiovascular research, which we recently reviewed 
			(<a href="http://circres.ahajournals.org/content/122/3/399.long"><span class="ptHyperlink">Link</span></a>). 
</p>
			
			<p class="ptNormal">
			In last few years, we have developed a number of bioinformatics tools:  
			
			<ul>
				<li>“<b>ANGIOGENES</b>” 
  					(<a href="http://angiogenes.kbrin.louisville.edu"><span class="ptHyperlink">Link</span></a>) 
					for expression profiles of transcripts in endothelial cells</li>
				
  				<li>“<b>C-It</b>” 
  					(<a href="http://c-it.mpi-bn.mpg.de/"><span class="ptHyperlink">Link</span></a>) 
					for evolutionary-conserved, tissue-enriched, uncharacterized genes</li>
				
				<li>“<b>C-It-Loci</b>” 
  					(<a href="http://c-it-loci.kbrin.louisville.edu"><span class="ptHyperlink">Link</span></a>) 
					for evolutionary-conserved, tissue-enriched, protein-coding genes and lncRNAs</li>
					
  				<li>“<b>Exon Array Analyzer</b> (<b>EAA</b>)” 
  					(<a href="http://eaa.mpi-bn.mpg.de/"><span class="ptHyperlink">Link</span></a>) 
  					for analyzing GeneChip Exon 1.0 ST Arrays (Affymetrix, Inc.)</li>
  					
				<li>“<b>Gene Array Analyzer</b> (<b>GAA</b>)” 
					(<a href="http://gaa.mpi-bn.mpg.de/"><span class="ptHyperlink">Link</span></a>) 
					for analyzing GeneChip Gene 1.0 ST Arrays (Affymetrix, Inc.)</li>
					
				<li>“<b>noncoder</b>” (<a href="http://noncoder.mpi-bn.mpg.de/"><span class="ptHyperlink">Link</span></a>) 
					for quantifying the expression changes of lncRNAs and protein-coding genes
					from GeneChip Exon 1.0 ST Arrays (Affymetrix, Inc.)</li>
				
				<li>“<b>RenalDB</b>” (<a href="http://renaldb.kbrin.louisville.edu/"><span class="ptHyperlink">Link</span></a>) for <i>in silico</i> screening of enriched/specific transcripts of humans, mice, and zebrafish with respect to nephrotic tissues and cells, developmental stages, and other metadata.
				
				<li>“<b>RNAEditor</b>” (<a href="http://rnaeditor.uni-frankfurt.de/"><span class="ptHyperlink">Link</span></a>) 
					for analyzing RNA editing sites and islands from RNA-seq data</li>
					
				<li>“<b>SNP4Disease</b>” 
  					(<a href="http://snp4disease.mpi-bn.mpg.de/"><span class="ptHyperlink">Link</span></a>) 
  					for disease-related and/or -suspected SNPs</li>
  					
				<li>“<b>UGAHash</b>” 
  					(<a href="http://ugahash.kbrin.louisville.edu"><span class="ptHyperlink">Link</span></a>) 
					for mapping human genomic features among various bioinformatics databases and versions of databases</li>
			</ul>
			
			<p class="ptNormal">
			Using the above bioinformatics tools combined with biological experiments, our research goal is to uncover
			the functions of miRNAs and lncRNAs as well as RNA modifications.
			</p>
			
			<ul class="contacts">
				<li><a href="http://cv2i.org">Cardiovascular Innovation Institute</a></li>
				</ul>
		</div>


</div>
  
  <footer>
   <ul>
   <li><a href="mailto:heart.lncrna@gmail.com">E-mail</a></li>
   <li><a href="http://cv2i.org/investigators/shizuka-uchida/">Cardiovascular Innovation Institute</a></li>
   </ul>
  </footer>
  </body>

</html>

